data/secscan_model/secscan_v4_model.py

```diff
@@ -195,7 +197,7 @@ def needs_reindexing_query(indexer_hash):
             yield_random_entries(not_indexed_query, Manifest.id, batch_size, max_id, min_id,),
             yield_random_entries(index_error_query, Manifest.id, batch_size, max_id, min_id,),
             yield_random_entries(
-                lambda: needs_reindexing_query(indexer_state),
+                lambda: needs_reindexing_query(indexer_state.get("state", "")),
```

Comment:
I'd prefer if the `.get` occurred at the API level

Reply:
All the rest of our methods return the full JSON. If we decide this endpoint needs more fields, then the secscan model will be able to use those.

Reply:
Okay.